core-services/prow/02_config/_config.yaml

```diff
@@ -641,7 +641,6 @@ tide:
     - release-4.2
     - release-4.4
     - release-4.3
-    - release-4.5
```

Comment:
Not this one. `ocs-operator` is AFAIK not involved in openshift processes yet, so we do not need to touch their queries